test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+RUN touch /sti-fake/src/index.html
+
+ADD scripts/.sti/bin/ /tmp/src/	
```

Comment:
First of all if you're adding those scripts inside the image, you should point S2I where to look for them with:
`LABEL io.openshift.s2i.scripts-url=image:///tmp/scripts/`. Or not to copy them here since the base image (`sti_test/sti-fake`) has `LABEL io.openshift.s2i.scripts-url=http://127.0.0.1:23456/.sti/bin`, which means the build will download the scripts from there.